util/testutil/shell.go

```diff
@@ -141,6 +142,42 @@ func (m *RemoteSnapshotMonitor) CheckAllRemoteSnapshots(t *testing.T) {
 	}
 }
 
+// FatalMonitor scans logs streamed from the specified io.Reader for fatal errors during startup
+func FatalMonitor(r shell.Reporter, stdout, stderr io.Reader) {
+	errs := make(chan error, 1)
+	go ScanFatalLog(io.TeeReader(stdout, r.Stdout()), errs)
+	go ScanFatalLog(io.TeeReader(stderr, r.Stderr()), errs)
+	err := <-errs
+	if err != nil {
+		r.Errorf(err.Error())
+	}
+}
+
+type LevelLogLine struct {
```

Comment:
same for the struct.

Reply:
This one is exported intentionally, in expectation of #520 and later new monitors using it.